pkg/virt-controller/watch/vmi_test.go

```diff
@@ -3260,3 +3367,17 @@ func newMultusNetwork(name, networkName string) virtv1.Network {
 		},
 	}
 }
+
+func newVMIWithOneNetwork(vmi *virtv1.VirtualMachineInstance, networkName string, ifaceName string) *virtv1.VirtualMachineInstance {
```

Comment:
The name hints as if it creates a new VMI, but it does not.
Based on the name, it appends a new network, so consider `appendNetworkToVMI`.

Also, please help the reader understand what info is passed into the function.
It will be much clearer if it will be `name` and `netAttachDefName`.

Reply:
I don't know what more to tell you. Let me try with an image, since people wiser than me say a picture is worth a thousand words. 

![image](https://user-images.githubusercontent.com/39051226/217774212-418cb4e2-2c88-462e-a9a7-e623e03ce441.png)

How can `networkName` be confusing ? Its **literally** the name of the attribute in the API.

Reply:
Renamed the function accordingly.

Reply:
This seems to repeat all the time and we fail to sync here.
`networkName` can mean two things:
- The name of the network in the spec, as we have networks, each entry is a network, and the attribute named `name`.
- The name of the NAD.

If someone messed up the naming 7 years ago, I do not feel obligated to keep confusing the code readers and continue this bad naming. Functions are abstractions and are a great place to be clear in the name, improving what was done in the past). This is true with manifests and with libraries which have bad API.

I seriosly do not understand how one cannot be confused by this. If I only read the code that this function is called from, I have no idea what this field means unless I look inside and read the code. And the whole point of having functions as abstractions is not to require the reader to go and read its content.

Reply:
Done.